.gitlab-ci.yml

```diff
@@ -1,4 +1,4 @@
-image: nkx1231/root6-geant4-garfield:0.6
+image: lobis/root-geant4-garfieldpp
```

Comment:
I still don't know why we have to switch the docker image

Reply:
The code was not compiling with your docker image ...

https://gitlab.cern.ch/rest-for-physics/framework/-/jobs/13162589

After updating the image it would compile again

https://gitlab.cern.ch/rest-for-physics/framework/-/commit/a489ac3e29ead42e1c86d21f8b166ad7453017bc/pipelines?ref=development

Reply:
Of course, I believe we could make it work with both images.

Doing a quick search I did not find how to run both images in gitlab in parallel. I.e. launching both images in two pipelines when we push to

Reply:
Is it because the previous updated c++14 compilation flag?

It seems not a big error to fix.

Reply:
> Doing a quick search I did not find how to run both images in gitlab in parallel.

Maybe that solve the problem

https://stackoverflow.com/questions/33387622/multiple-docker-images-in-gitlab-ci-yml